Docs/install/install_host.rst

```diff
@@ -94,55 +113,57 @@ Install GPU packages for PyTorch 2.1 or PyTorch 1.13 or ONNX or TensorFlow
     dpkg -i cuda-keyring_1.0-1_all.deb
     apt-get update
 
-Install AIMET packages
-~~~~~~~~~~~~~~~~~~~~~~~
 
-From PyPI
-=========
+Installing AIMET
+================
 
-The default AIMET Torch GPU variant may be installed from PyPI as follows:
-    - Go to https://pypi.org/project/aimet-torch
-    - Browse the Requirements section of each Release to identify the version you wish to install. Following are some tips:
-        - For Pytorch 2.2.2 GPU with CUDA 12.1, use aimet-torch>=1.32.2
-        - For Pytorch 2.1.2 GPU with CUDA 12.1, use aimet-torch==1.32.1.post1
-        - For PyTorch 1.13 GPU with CUDA 11.7, use aimet-torch==1.31.2
+**Choose your AIMET variant.**
 
-Run the following commands to install the package (prepend with "sudo" and/or package version as needed):
+Based on your machine learning framework and GPU preference, choose one of the install procedures below.
 
-.. code-block:: bash
+:ref:`1. Installing AIMET for PyTorch <man-install-torch>`
+
+:ref:`2. Installing AIMET for TensorFlow <man-install-tf>`
+
+:ref:`3. Installing AIMET for ONNX <man-install-onnx>`
+
+.. _man-install-torch:
 
-    apt-get install liblapacke -y
```

Comment:
Can you please remind me where these steps moved to?

Reply:
The install procedures follow below in the order they're listed (starting immediately below with PyTorch). Presumably in the Install rewrite they'll be on separate pages or in tabs to avoid the confusion of listing them sequentially.

It looks like I forgot to include the liblapacke install instruction on this page. Does this apply only to the PyTorch PyPI install, or generally to all the installs? I'll modify here and on the main install page as needed.

I've also omitted the PyPI quick install instructions on this page. This exists on the main install page, and this page gives complete instructions for the wheel install. Is it necessary to include the PyPI install here?

Reply:
"Does this apply only to the PyTorch PyPI install, or generally to all the installs?" Good question - Let's add it to all installs for now. We can remove from the others later if they are not needed

Reply:
OK, I've added the liblapacke dependency to the prereqs, so it's required for all the install procedures.